rules/windows/defense_evasion_network_connection_from_windows_binary.toml

```
@@ -17,10 +17,64 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Unusual Network Activity from a Windows System Binary"
+note = """## Triage and analysis
+
+### Investigating Unusual Network Activity from a Windows System Binary
+
+Attackers can abuse certain trusted developer utilities to proxy the execution of malicious payloads. Since these utilities are usually signed, they can bypass the security controls that were put in place to prevent or detect direct execution.
+
+This rule identifies network connections done by trusted developer utilities, which can indicate abuse to execute payloads or process masquerading.
```

Nit word choice: maybe "established" or "created"?
```suggestion
This rule identifies network connections established by trusted developer utilities, which can indicate abuse to execute payloads or process masquerading.
```